.editorconfig

```diff
@@ -104,7 +104,7 @@ csharp_style_conditional_delegate_call = true:suggestion
 # Modifier preferences
 csharp_preferred_modifier_order = public,private,protected,internal,file,static,extern,new,virtual,abstract,sealed,override,readonly,unsafe,required,volatile,async:suggestion
 # Expression-level preferences
-csharp_prefer_braces = false:none
+csharp_prefer_braces = true:suggestion
```

Comment:
You make it a suggestion, but fix all/a lot? Sounds like it is mandatory :smile: 
(I am fine either way)

Reply:
I'd say make it mandatory. Suggestions are neither this nor that. Make it clear! :)

Reply:
> You make it a suggestion, but fix all

The Code Cleanup applies the fix if the rule is `suggestion` or higher. Not sure what would happen if there were a [`silent`](https://learn.microsoft.com/en-us/dotnet/fundamentals/code-analysis/configuration-options#severity-level) rule, but Code Cleanup doesn't apply fixes if the severity is `none`.

In other words, the severity `suggestion` means that Code Cleanup applies the fix, and the rule is non-mandatory for the dev.